Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gem 'capistrano-s3'
+gem 'dotenv-rails'
```

Comment:
That would be interesting. I have a hard time imagining that for Ruby, though, because setting up a version of Ruby is pretty painful and unique to different configurations.